src/main/java/org/mule/extension/runtime/ExpirationPolicy.java

```diff
@@ -0,0 +1,35 @@
+package org.mule.extension.runtime;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * A policy around when should a given instances be considered elegible for expiration.
```

Comment:
In your previous comment you describe an example where something is expired after 1 minute, but the platform checks expirations every 5 minutes. 
Under this scenario, my previous question is that at the 5th minute, can the platform decide to ignore the expiration of that item?
If the answer is no, the the item is not elegible for expiration